webdriver/tests/support/fixtures_bidi.py

```diff
@@ -214,8 +215,10 @@ async def assert_pdf_dimensions(pdf, expected_dimensions):
         width, height = png_dimensions(png)
 
         # account for potential rounding errors
-        assert (height - 1) <= cm_to_px(expected_dimensions["height"]) <= (height + 1)
-        assert (width - 1) <= cm_to_px(expected_dimensions["width"]) <= (width + 1)
+        assert (
```

Comment:
Please get rid of those extra formatting changes which are not relevant to this particular PR. There are some others at the top as well.